Исправление дублирования операции в истории выдач сотрудника.

Katetim32 opened the pull request.

Gankov commented:
Я думаю что это не правильное решение.
Во первых, зачем эта строчка сейчас? Почему выбран Skip? Ведь в таком виде есть эта строчка или нет, ничего не изменится. Тогда можно было бы ее просто удалить.
Но ведь изначальная логика добавления сюда этой строчки была для ускорения загрузки списка операция, Такое решение замедлит это.
Хотя оно изначально было не правильное, потому как баркоды то сами подгружались все равно отдельным запросом, и поэтому ускорение толком не работало. Тут еще и баркоды надо фетчить.
И кстати не понимаю почему при таком решении хебирнетй дублирует строки, по идеи не должен так как коллекция всего одна. 
Но если уж он так делает, более правильным решением было бы вынесение запроса штрих кодов в отдельным подзапрос в через Future

Gankov committed:
2.9.2

Gankov committed:
Merge branch 'release/2.8'

Katetim32 committed:
Merge remote-tracking branch 'origin/master' into feature/I-711/T-2174

Katetim32 committed:
Добавление кнопки для добавления списания с дежурной нормы.

Gankov committed:
Исправление чендж лога.

Gankov committed:
Добавлен ответ для поставок.

Katetim32 committed:
Добавила фильтр для журнала.

Gankov committed:
Добавлены отсутсвующие правила удаления для планируемой поставки.

Katetim32 committed:
Добавила ViewModel журнала остатков по дежурным нормам.

И также добавила статический метод для подсчета износа в доменную модель операции выдачи по деж. норме.

Gankov committed:
Добавлена документация по планируемым поставкам.

Katetim32 committed:
Добавление журнала.

Katetim32 committed:
Добавление необходимых методов, свойств и перегрузка конструктора в доменных моделях.

Katetim32 committed:
Добавила метод для добавления из дежурной нормы во вьюмодель списания.

Gankov committed:
Документация по документу выдачи дежурных СИЗ.

Katetim32 committed:
Обработала кнопку добавления, исправила ошибку.

Katetim32 committed:
Добавление операции списания с дежурной нормы в строки док-та списания.

Katetim32 committed:
Добавление мапинга.

Katetim32 committed:
Исправление ошибки, добавление апдейта и нужных условий.

Gankov committed:
* В модуль стирки добавлены статусы: "Доставка в прачечную", "В химчистке", "Доставка в постамат выдачи".

Katetim32 committed:
Добавление проверки количества.

Katetim32 committed:
Merge remote-tracking branch 'origin/master' into feature/I-711/T-2174

Gankov committed:
Merge pull request #610 from QualitySolution/feature/I-711/T-2174

Добавление списания с дежурной нормы.

Gankov committed:
Merge branch 'release/2.8'

Gankov committed:
Убрал лишний код.

Gankov committed:
чендж лог

Katetim32 committed:
Merge remote-tracking branch 'origin/master' into feature/I-711/T-2213

Katetim32 committed:
Добавила операции списания с дежурной нормы в печатную форму.

Katetim32 committed:
Merge remote-tracking branch 'origin/master' into feature/I-711/T-2212

Katetim32 committed:
Внесла изменения в sql-запрос.

Gankov committed:
Merge pull request #612 from QualitySolution/feature/I-711/T-2213

Добавила операции списания с дежурной нормы в печатную форму акта списания.

Katetim32 committed:
Merge remote-tracking branch 'origin/master' into feature/I-711/T-2212

Katetim32 committed:
Добавила во вьюшку возврата дежурную норму. В enum добавила возврат с дежурной нормы.

Katetim32 committed:
Изменила видимость во вьюшке для разных видов возврата.

Katetim32 committed:
Изменила вьюшку, сделала один элемент enum Возврат.

Katetim32 committed:
Убрала ненужное и сделала возможным использовать фильтр по сотрудникам и складам в остатках.

Katetim32 committed:
Добавила скрипт 2.10, изменения в строках документа возврата.

Katetim32 committed:
Добавление в доменную модель и мапинг.

Gankov committed:
В отзывах на продукцию добавлена возможность открыть карточку сотрудника.

Gankov committed:
В журнал сотрудников добавлен поиск по полностью введенному телефону или email адресу.

Gankov committed:
Задокументирован поиск по сотрудника.

Gankov committed:
Починил открытие url на линуксе.

Gankov committed:
Доработка документации по сотрудникам.

Gankov committed:
Исправление ошибок в документации + добавил якорей

+ Хелпер по ссылка на документацию.

Gankov committed:
Merge branch 'release/2.8'

Gankov committed:
Исправил версию в документации.

Gankov committed:
Добавлены ссылки на документацию во все диалоги.

Gankov committed:
Доработка документации по постаматам.

Gankov committed:
Повесил документацию почти на все журналы.

Gankov committed:
Исправил опечатку в названии класса.

Gankov committed:
Добавлены ссылку на документацию для всех отчетов имеющих параметры.

Gankov committed:
Добавил ченджлог

SpaceTranquility committed:
Исправление бага с удалением документа постомата

Gankov committed:
Merge pull request #613 from QualitySolution/fix/27027

Исправление бага с удалением документа постомата

Katetim32 committed:
Изменения за 28.03

Перенос сотрудника из док-та в строки док-та.

Gankov committed:
Ченджлог

Gankov committed:
Merge remote-tracking branch 'origin/master'

Katetim32 committed:
Добавление с дежурной нормы

Gankov committed:
Добавил права на удаление объектов пока в устаревшем виде.

SpaceTranquility committed:
Базовый модуль выбора списком

SpaceTranquility committed:
Перевод на новый виджет списка

Gankov committed:
Добавлена настройка даты запрета редактирования.

Gankov committed:
Исправлена опечатка в скрипте создания

Похоже из-за этого не работали тесты.

Gankov committed:
Убираем из номеклатурного номера незначащие символы, такие как перенос строки, в конце текста.

Gankov committed:
Merge remote-tracking branch 'origin/master'

Gankov committed:
Обновил сабмодуль

Gankov committed:
Merge branch 'feature/28569'

Gankov committed:
В журнал штрихкодов добавлен поиск по табельному номуре.

Gankov committed:
Обновил описание мобильного приложения.

Gankov committed:
Представлен пример реализации даты запрета редактирования в документе поступления.

Gankov committed:
Исправлено заполнение параметра даты запреты редактирования.

Gankov committed:
Изменения в библиотеках касающиеся прав.

Gankov committed:
Скрытие кнопки сохранить в диалогах открытых только для просмотра.

Gankov committed:
Дата запрета редактирования добавлена в документацию.

Gankov committed:
Удалил видимо забытый Ваней виджет.

Gankov committed:
Заменил виджет в настройках даты запреты редактирования на тот, который имеет кнопку очистки.

SpaceTranquility committed:
Добавление интерфейса прав в складские документы и ведомость

SpaceTranquility committed:
Merge remote-tracking branch 'origin/master' into feature/29011.2

Katetim32 committed:
Исправила ошибку, добавила метод для проверки на количество при добавлении.

Katetim32 committed:
Переделала условие.

Katetim32 committed:
Merge remote-tracking branch 'origin/master' into feature/I-711/T-2212

Katetim32 committed:
Удалила комментарий.

Katetim32 committed:
Исправления.

Katetim32 committed:
Исправила TODO

Katetim32 committed:
Подправила TODO

Gankov committed:
Merge pull request #616 from QualitySolution/feature/29011.2

Права в складских документах.

Gankov committed:
Merge branch 'master' into feature/I-711/T-2212

Gankov committed:
Merge pull request #615 from QualitySolution/feature/I-711/T-2212

Добавление возврата с дежурной нормы.

Gankov committed:
2.10 -начало подготовки

Gankov committed:
Перенес sql скрипты из 2.9.3 в 2.10

Gankov committed:
Исправлена сборка после мержа

Gankov committed:
Исправление сборки тестов

SpaceTranquility committed:
Не запрещаем редачить несвязанную ведомость

SpaceTranquility committed:
Исправление списания

SpaceTranquility committed:
Добавил причину расхождения

SpaceTranquility committed:
Исправил дату

SpaceTranquility committed:
ПДополнил лог

Gankov committed:
Исправление скриптов работы с базой.

SpaceTranquility committed:
Добавил статусы

Gankov committed:
Merge pull request #618 from QualitySolution/fix/i-1646

Исправил дату

Gankov committed:
Обновил чендж лог.

Gankov committed:
Заменил версию в документации

Gankov committed:
Добавлено описание ссылок на документацию.

Gankov committed:
Исправлена версия документации

Gankov committed:
Исправил отображение откуда списано.

Gankov committed:
Исправлена печатная форма на возврат.

Gankov committed:
Исправил еще 2 печатные формы ведомостей.

SpaceTranquility committed:
Табельный в забборе из постомата

Gankov committed:
Merge pull request #619 from QualitySolution/feature/29435

Табельный в забборе из постомата

SpaceTranquility committed:
Разделение колонок.

Gankov committed:
Задокументирован возврат

Gankov committed:
Списание по дежурной норме добавлено в документацию.

Gankov committed:
Исправил расположение столбца

Gankov committed:
Исправлено заполнение поля с дежурной нормой.

Gankov committed:
Добавлено отображение дежурных норм в журнале для документа списания и возврата.

Gankov committed:
Исправил конфигурацию удаления по тестам.

Gankov committed:
Исправил тесты.

SpaceTranquility committed:
Добавил ссылку на поставку.

Gankov committed:
Merge branch 'release/2.8'

SpaceTranquility committed:
Добавил в поставку поле и сделал заполнение поступления

SpaceTranquility committed:
Сделал Учёт поступлений

SpaceTranquility committed:
Фильтр поставок и правка вьюшек

Gankov committed:
Разделил логику прогнозирования на 2 варианта по номенклатуре нормы и по складской.

SpaceTranquility committed:
Реализовал фльтр, добавил "частично"

Gankov committed:
Добавлена основная реализация прогнозирования по номенклатуре.

Gankov committed:
Удалил уже не актуальные каталоги из скрипта.

Gankov committed:
Исправил ошибку случайно переместил не туда виью модель

Gankov committed:
Реализовал фильтрацию по складской номеклатуре, по номеклатуре нормы не до делал.

SpaceTranquility committed:
разделил статтусы, добавил 2 ограничения + сепараторы

Gankov committed:
Скрыл дату запрета редактирования из однопользовательской версии.

Gankov committed:
Добавление прав пользователя на изменение настроек учета.

SpaceTranquility committed:
Сделал поставлено хранимым, и сделал расчёт

Gankov committed:
Исправил работу теста.

SpaceTranquility committed:
Настроил цвета

SpaceTranquility committed:
Merge remote-tracking branch 'origin/master' into featere/i-1628_1625

Gankov committed:
Merge branch 'refs/heads/master' into feature/forecastmodification

SpaceTranquility committed:
Починили легенду

SpaceTranquility committed:
Доработал удаление

SpaceTranquility committed:
Добавил емаил в настройки пользователя. Запрет на сохранение не ставлю

Katetim32 committed:
Добавление печатной формы документа планируемой поставки (закупки).

SpaceTranquility committed:
Добавил виджет Email И отправку закупке

Gankov committed:
Merge pull request #622 from QualitySolution/feature/I-1630

Добавление печатной формы документа планируемой поставки (закупки).

SpaceTranquility committed:
Merge remote-tracking branch 'origin/featere/i-1628_1625' into featere/i-1628_1625

SpaceTranquility committed:
Испраавления

SpaceTranquility committed:
Merge remote-tracking branch 'origin/feature/forecastmodification' into featere/i-1628_1625

Gankov committed:
Вроде доделана фильтрация по номенклатуре.

SpaceTranquility committed:
Merge remote-tracking branch 'origin/feature/forecastmodification' into featere/i-1628_1625

Gankov committed:
Добавлено колонка с ценой в экспорт прогноза.

SpaceTranquility committed:
Создание поставки из Прогноза

SpaceTranquility committed:
Рефакторинг фильтра баланса сотрудников

SpaceTranquility committed:
Не считается износ в возврате

SpaceTranquility committed:
Переключил библиотеки + лог

Gankov committed:
Merge pull request #623 from QualitySolution/fix/29907

Ошибка с износом при возврате

SpaceTranquility committed:
Дополнительный оттчёт по обслуживанию одежды

Gankov committed:
Merge pull request #624 from QualitySolution/frature/i-1703_service_report

Дополнительный оттчёт по обслуживанию одежды

Gankov committed:
Чтобы собирать предварительную версию 2.10.1

SpaceTranquility committed:
Отчёт с метриками стирки

SpaceTranquility committed:
Отчёт по количеству обращений

SpaceTranquility committed:
Лог

Gankov committed:
Merge pull request #625 from QualitySolution/feature/i-1720_claim_metric

Отчёт с метриками стирки

Gankov committed:
Исправление тестов

Gankov committed:
Merge remote-tracking branch 'origin/master'

Gankov committed:
Исправил работу теста.

SpaceTranquility committed:
Отчёт по использованию постаматов

SpaceTranquility committed:
Лог

Gankov committed:
Merge pull request #626 from QualitySolution/feature/i-1708_postama_report

Отчёт по использованию постаматов

SpaceTranquility committed:
Добавил дату в отчёт

Gankov committed:
Merge pull request #627 from QualitySolution/fix/report_period

Добавил дату в отчёт

SpaceTranquility committed:
Группировка по подразделениям

SpaceTranquility committed:
Убрал глючный отчёт, реализовал в другом

Gankov committed:
Удалил дубль объявления

Gankov committed:
Исправил опечатку. Улучшил внешний вид окна отправки сообщения.

Gankov committed:
Merge branch 'featere/i-1628_1625'

SpaceTranquility committed:
Исправление справки по размерам

Gankov committed:
Исправил ошибки по тестам

Gankov committed:
Исправил конфликты мержа.

Были поломаны виджеты.

Katetim32 committed:
Merge remote-tracking branch 'origin/master' into feature/I-1630

Gankov committed:
Исправление скриптов обновления базы.

Katetim32 committed:
Добавление rdl

Katetim32 committed:
Добавление вьюшки и вьюмодели.

Gankov committed:
2.10.1

Katetim32 committed:
Merge branch 'master' into feature/I-1630

Gankov committed:
Добавлен чендлог по планируемой поставке.

Gankov committed:
Merge pull request #628 from QualitySolution/feature/27080_service_report

Группировка по подразделениям в отчёте по обслуживанию

SpaceTranquility committed:
Добавил ограничение по фичам

SpaceTranquility committed:
Правка

SpaceTranquility committed:
Исправление фильтра

SpaceTranquility committed:
Фикс группировки для возврата

Gankov committed:
Реализовал отображение заказанного количества в прогнозе.

Gankov committed:
Добавил колонки со следующей поставкой в выгрузку прогнозирования склада.

SpaceTranquility committed:
Фильтр по собственнику в складской ведомости

Gankov committed:
Добавлен режим отображение всего подходящего.

Gankov committed:
Merge pull request #629 from QualitySolution/feature/27080_service_report

Добавил ограничение по фичам

SpaceTranquility committed:
Правка журнала документов

Gankov committed:
Merge pull request #630 from QualitySolution/fix/30529_stock_documents

Фикс группировки для возврата

Gankov committed:
Merge pull request #631 from QualitySolution/feature/30566_stock_report

Фильтр по собственнику в складской ведомости

SpaceTranquility committed:
Добавил в отчёт колонку со статусом

Gankov committed:
Merge pull request #632 from QualitySolution/feature/30755_service_report

Добавил в отчёт колонку со статусом

Gankov committed:
Исправлена ошибка в прогнозировании склада.

Gankov committed:
Исправлена ошибка в окраске

Gankov committed:
Исправлено пропадание зимы в прогнозе который делается в летний период.

Gankov committed:
Удалил дебажный код.

Gankov committed:
Добавлен тест реального бага с тем что у нас в прогнозировании пропадали долги если оно попадала в период не выдачи.

Gankov committed:
Исправление опечаток

Gankov committed:
Добавил обновление цифр поставки если поставка открыта в момент ввода новой накладной.

Gankov committed:
Добавлен удобный способ скопировать значения запрошенного в заказаное.

SpaceTranquility committed:
Проверка уже занятых ячеек

Katetim32 committed:
Merge remote-tracking branch 'origin/master' into feature/I-1630

Gankov committed:
Добавлено отображение в прогнозе кто именно должен придти за выдачей.

Gankov committed:
Merge branch 'release/2.8'

Katetim32 committed:
Merge branch 'master' into feature/I-1630

Katetim32 committed:
Запрос для названия отчета. Обработка вьюмодели.

Katetim32 committed:
Откат изменений в стетике, удаление вьюшки.

Были ошибки при запуске в monodevelop

Katetim32 committed:
Добавление новой вьюшки.

Katetim32 committed:
Добавление в меню

Katetim32 committed:
Правки rdl и её добавление

Katetim32 committed:
Добавление артикула в фильтр номенклатур.

Katetim32 committed:
Правки в запросе для корректного отображения даты в названии отчета.

Katetim32 committed:
Поменяла дату создания на период поставок в названии отчета.

Katetim32 committed:
Исправление сортировки

Katetim32 committed:
Правки в основной части запроса.

Katetim32 committed:
Переопределение title для фильтров.

Katetim32 committed:
Переименовала "предполагаемые" на "планируемые" поставки.

Katetim32 committed:
Удалила строку подключения в rdl.

SpaceTranquility committed:
Правки по пулреквесту

Gankov committed:
Merge pull request #636 from QualitySolution/featurr/i_1773_shipment

Правки по пулреквесту

Gankov committed:
Merge pull request #635 from QualitySolution/feature/I-1630

Добавление отчета по планируемым поставкам (закупкам).

Gankov committed:
Merge branch 'release/2.8'

Gankov committed:
Добавил изменения  чендждлог

Katetim32 committed:
Правки в запросе.

Чтобы периоды, где отсутствует значение, не отображались.

Katetim32 committed:
Правки в фильтре документов поставок.

Исправление ошибки и добавление статуса поставки.

Gankov committed:
Merge pull request #637 from QualitySolution/feature/I-1630-fix

Исправления в отчете по планируемым поставкам.

SpaceTranquility committed:
Поменял проверку на дату выдачи

Gankov committed:
Merge pull request #633 from QualitySolution/fix/i-1762

Проверка уже занятых ячеек при сохранении документа постомата.

Katetim32 committed:
Добавление копирования дежурной нормы.

Gankov committed:
Merge pull request #638 from QualitySolution/feature/28216

Добавление копирования дежурной нормы.

Gankov committed:
Добавил в чендждолг

Gankov committed:
Merge branch 'release/2.8'

Gankov committed:
Привел ченлждлог к более красивому виду

Gankov committed:
Исправлена опечатка

использовалась русская буква вместо английской

Gankov committed:
В планируемую запупку добавлена колонка недоставлено.

Gankov committed:
Еще исправление опечаток в коде

Gankov committed:
Добавлено засеривание кнопок добавить и удалить если потребности нельзя редактировать.

Gankov committed:
Добавил документацию на новый процесс планирования поставок

Gankov committed:
Добавлено документирование нового функционала в прогнозировании.

Katetim32 committed:
Убрала цвет в ячейке.

Gankov committed:
Merge pull request #640 from QualitySolution/feature/fix-clothingservice

Убрала цвет в ячейке.

Gankov committed:
Добавлена документация по новым отчетам

Gankov committed:
Merge remote-tracking branch 'origin/master'

Gankov committed:
Изменил дату релиза

Katetim32 committed:
Merge branch 'master' into feature/25700

Gankov commented:
И что? Как это исправит проблему в 2.8? Мы же не можем это в 2.8 залить. Она тогда не 2.8 будет уже не 2.8.
Если это исправлено уже в мастере, перенеси пожалуйста исправление вручную. сделай отдельный пул реквест. Этот уже испорчен.
